docs/non-normative/code-generation.md

```diff
@@ -1,9 +1,4 @@
-<!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Generating Semantic Convention libraries
-# Renamed from: semantic_conventions_code_generation
```

Comment:
Does this "renamed from" do anything in the OTel website?

Reply:
No, it is a comment.